### PR TITLE
feat: create get student accepted lessons route and get student lessons by status

### PR DESCRIPTION
**Pull request by RCAS2021**
CHANGES 📝

- Add get student confirmed lesson requests route
- Add getStudentPendingLessons, getStudentConfirmedLessons and getStudentLessonsByStatus in student controller
- Update findPendingLessonByStudentId to findStudentLessonsByStatus to find lessons based on status in student repository
- Update getPendingLessonByStudentId to getStudentLessonsByStatus to get lessons based on status in student service
- Implement lesson request formatting using formatter function

CARDS 📚

[Lesson Request](https://trello.com/c/gRAwxd2p/36-get-aulas-confirmadas-aluno)

NOTE 📝

Branch "feature/student-confirmed-lessons" -> "feature/update-student-lesson"

DETAILS 📝
Get confirmed student lessons:
{9B59B3EE-EE3A-423E-B5F1-2B2DEAAFA172}

Get pending student lessons:
{5A1D9B82-AD79-4867-8F01-C3D38E719D8A}

### CODE REVIEW

File: src/routes.ts [hunk 1]
```diff
@@ -72,7 +72,13 @@ router.get(
 router.get(
   '/api/get/student-pending/:id',
   authMiddleware(),
-  new StudentController().getPendingLessonByStudentId
+  new StudentController().getStudentPendingLessons
+);
+
+router.get(
+  '/api/get/student-confirmed-lessons/:id',
```
Comment (G4brielSilva) [inline src/routes.ts]: Faz sentido ter get na rota sendo que o método já é get?

**Review by G4brielSilva — CHANGES_REQUESTED** [review]

File: src/routes.ts [hunk 2]
```diff
@@ -28,6 +28,11 @@ router.post(
 );
 
 router.get('/api/get/tutor', authMiddleware(), new TutorController().getAll);
+router.get(
+  '/api/get/tutor/:id',
```
Comment (G4brielSilva) [inline src/routes.ts:32]: mais rotas get com get no path, corrija

File: src/routes.ts [hunk 3]
```diff
@@ -72,7 +77,13 @@ router.get(
 router.get(
   '/api/get/student-pending/:id',
```
Comment (G4brielSilva) [inline src/routes.ts:78]: essa também

**Review by G4brielSilva — CHANGES_REQUESTED** [review]

**Comment by samuelcamilocosta** [comment]
PR fechado por não cumprir os requisitos do gitflow e por estar desatualizado, orientei o Rafael com o caminho necessário para abrir o próximo PR para a develop

**Pull request closed without merging**